dds/DCPS/RapidJsonTypeConverter.h

```diff
@@ -0,0 +1,32 @@
+#ifndef OPENDDS_DCPS_RAPIDJSONTYPECONVERTER_H
+#define OPENDDS_DCPS_RAPIDJSONTYPECONVERTER_H
+
+#include "rapidjson/document.h"
+
+OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
+
+namespace DDS {
+  class DataWriter;
+}
+
+namespace OpenDDS {
+namespace DCPS {
+  class RapidJsonTypeConverter {
+  public:
+    virtual ~RapidJsonTypeConverter() {}
+
+    virtual void toRapidJson(const void* source, rapidjson::Value& dst, rapidjson::Value::AllocatorType& alloc) const = 0;
+    virtual void* fromRapidJson(const rapidjson::Value& source) const = 0;
+    virtual void deleteFromRapidJsonResult(void* obj) const = 0;
+
+    virtual DDS::InstanceHandle_t register_instance_helper(DDS::DataWriter* dw, const void* data) const = 0;
```

Comment:
Is the RapidJson support (in this iteration) going to support read/write operations like V8 does, or only the convert to/from operations?

Reply:
I'm not sure how it would support read / write, as it's not a JS engine like V8, just a JSON parser.

Reply:
I agree, so based on that fact can we remove the writer-related methods from this class?